intellij-plugin/src/main/kotlin/com/apollographql/ijplugin/telemetry/TelemetryService.kt

```diff
@@ -95,7 +94,7 @@ class TelemetryService(
 
   private val telemetryEventList: TelemetryEventList = TelemetryEventList()
 
-  private var projectLibraries: Set<Dependency> = emptySet()
+  private var projectLibraries: Set<MavenCoordinates> = emptySet()
 
   private val executor: ScheduledExecutorService = Executors.newSingleThreadScheduledExecutor()
```

Comment:
Is this a virtual thread if I run my IntelliJ on Java 21? 🤓

Reply:
Haha possibly .... but probably not 😁.